security_groups.tf

```diff
@@ -1,13 +1,14 @@
-resource "aws_security_group" "eq_alb" {
-  name        = "${var.env}-eq-alb"
-  description = "Allow access from the internet or WAF"
+resource "aws_security_group" "eq_alb_waf" {
+  name        = "${var.env}-eq-alb-waf"
```

Comment:
From our offline conversation, we need to re-work the naming:
The security group name should be the specific identifier for this security group e.g. `{env}-eq-alb-access-from-waf`, and the `Name` tag should be used to group the related security groups, set to something like `{env}-eq-ecs`